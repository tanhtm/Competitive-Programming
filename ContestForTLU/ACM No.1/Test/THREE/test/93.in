677214233000000000
