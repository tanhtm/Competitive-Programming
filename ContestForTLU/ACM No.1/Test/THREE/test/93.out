0 2 3 4 6 9 11 12 13 17 18 22 23 27 28 32 33 34 35 36 37 
5 8 10 15 16 19 20 21 24 29 30 31 38 
